code/crates/test/proto/transaction.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+import "consensus.proto";
+
+package test;
+
+// This is a transaction that is already accepted in a block.
+message Transaction
+{
+    message Dummy {
+        bytes bytes = 1;
+    }
+
+    message DeclareV0 {
+        Address sender = 1;
+        Signature signature = 3;
+        Hash class_hash = 4;
+    }
```

Comment:
We already have a dummy tx type so we can remove this I think, it's a starknet-specific tx type.